i18n/locales/fr-FR/dashboard/data.yml

```diff
@@ -476,10 +476,10 @@ fr:
         MC_2018_loops: 'Minecraft : Voyage Aquatique - Répéter jusqu''à'
         mc_2018_conditionals: 'Minecraft : Voyage Aquatique - Instructions conditionnelles'
         mc_2018_closing: 'Minecraft : Voyage Aquatique - Félicitations'
-        hoc_dance_warmup: Fête dansante - Échauffement
+        hoc_dance_warmup: Dance Party - Warm Up
```

Comment:
This seems to have reverted to English?

Reply:
I believe this was intentional; "Dance Party" is a brand and shouldn't be translated. @castro-jorge can you confirm?